test/bridge/testase.py

```diff
@@ -0,0 +1,261 @@
+        assert np.isclose(data.mult, 1)
+        assert np.isclose(data.temperature, 0)
+
+    def test_makecclib_retrieves_optimization(self):
+        """Ensure makecclib works with native ASE Atoms objects."""
+        h2 = Atoms("H2", positions=[[0, 0, 0], [0, 0, 0.7]])
+
+        # check whether converting back gives the expected data
+        data = cclib2ase.makecclib(h2)
+        assert np.allclose(data.atomcoords, [[[0, 0, 0], [0, 0, 0.7]]])
+        assert np.allclose(data.atomnos, [1, 1])
+        assert np.allclose(data.atommasses, [1.008, 1.008])
+        assert np.isclose(data.natom, 2)
+        assert np.isclose(data.charge, 0)
+        assert np.isclose(data.mult, 1)
+        assert np.isclose(data.temperature, 0)
+
+    def test_makease_works_with_openshells(self):
+        """Ensure makease works from parsed data for open-shell molecules."""
+        # make sure we can construct an open shell molecule
+        data = ccopen("data/ORCA/basicORCA4.2/dvb_sp_un.out").parse()
+        dvb_sp_un = cclib2ase.makease(
+            data.atomcoords[-1],
+            data.atomnos,
+            data.atomcharges["mulliken"],
+            data.atomspins["mulliken"],
+            data.atommasses,
+        )
+
+        # check whether converting back gives the expected data
+        ase_data = cclib2ase.makecclib(dvb_sp_un)
+        assert np.allclose(ase_data.atomcoords, [data.atomcoords[-1]])
+        assert np.allclose(ase_data.atomnos, data.atomnos)
+        assert np.allclose(
+            ase_data.atomcharges["mulliken"], data.atomcharges["mulliken"]
+        )
+        assert np.allclose(
+            ase_data.atomspins["mulliken"], data.atomspins["mulliken"]
+        )
+        assert np.allclose(ase_data.atommasses, data.atommasses)
+        assert np.isclose(ase_data.charge, data.charge)
+        assert np.isclose(ase_data.mult, data.mult)
+        assert np.isclose(ase_data.natom, len(data.atomnos))
+        assert np.isclose(ase_data.temperature, 0)
+
+        # make sure our object is compatible with ASE API
+        dvb_sp_un.calc = EMT(label="dvb_sp_un")  # not a serious calculator!
+
+        # converting back should give updated results
+        ase_data = cclib2ase.makecclib(dvb_sp_un)
+        assert np.allclose(ase_data.atomcoords, [data.atomcoords[-1]])
+        assert np.allclose(ase_data.atomnos, data.atomnos)
+        assert np.allclose(ase_data.atommasses, data.atommasses)
+        assert np.allclose(
+            ase_data.atomcharges["mulliken"], data.atomcharges["mulliken"]
+        )
+        assert np.allclose(
+            ase_data.atomspins["mulliken"], 0
+        )  # spin densities not supported but overwritten by EMT
+        assert np.isclose(ase_data.charge, data.charge)
+        assert np.isclose(ase_data.mult, 1)
+        assert np.isclose(ase_data.natom, len(data.atomnos))
+        assert np.isclose(ase_data.temperature, 0)
+        assert np.allclose(ase_data.scfenergies, [7.016800805424298])
```

Comment:
Can you comment that this value and the gradients came from the EMT calculation? I was confused by this at first.

Reply:
Done!